package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "superformula-backend-test",
+  "version": "1.0.0",
+  "description": "API with CRUD endpoints for user records",
+  "main": "code/main.js",
+  "dependencies": {
+    "boom": "7.2.0",
+    "config3": "1.0.3",
+    "hapi": "17.5.3",
+    "hapi-mongodb": "7.1.0",
+    "hapi-pino": "5.0.1",
+    "hapi-swagger": "^9.1.1",
+    "inert": "^5.1.0",
+    "joi": "13.6.0",
+    "joi-summarize": "1.1.0",
+    "process-title": "1.0.0",
+    "uuid": "3.3.2",
+    "vision": "^5.3.3"
+  },
+  "devDependencies": {
+    "eslint": "5.3.0",
+    "eslint-config-airbnb-base": "13.0.0",
+    "eslint-config-prettier": "2.9.0",
+    "eslint-formatter-comment": "2.0.0",
+    "eslint-plugin-import": "2.13.0",
+    "eslint-plugin-promise": "3.8.0",
+    "eslint-tap": "2.0.1",
+    "node-dev": "3.1.3",
+    "prettier": "1.14.2",
+    "prettier-eslint-cli": "4.7.1",
+    "supertest": "3.1.0",
+    "tap": "12.0.1"
+  },
+  "scripts": {
+    "test": "tap 'code/**/*-tap.js'",
+    "coverage": "tap --jobs-auto --coverage-report=html 'code/**/*-tap.js'",
+    "format": "prettier-eslint --write 'code/**/*.js'",
```

Comment:
Would be nice to run prettier on pre-commit and staged files using husky or a technique like this https://gist.github.com/primaryobjects/9ab8d2346aa58ef6959ad357bb835963

Reply:
This way there will be little to no chance of poorly formatting code making it's way into the repo.

Reply:
I'd be OK with that but I'd rather have it fail and abort the commit rather than running code formatting during a git hook and proceeding with a commit. I have found you can go too far with git hooks which tend to be project-specific custom behavior and git is a common tool that should behave mostly consistent across all projects in my opinion. I find it very jarring if I type a git command that does X in 99 projects and in this project it does X plus Y where Y is a very different thing in my mind (like pushing a branch or changing the source code). Thus I tend to not have nor want git hooks that alter source code. I leave the source code changing to the formatting script and git for committing the code as is.

Reply:
We can write contribution guidelines that can address this need to.  Many editors can run prettier and eslint on demand.

Thanks for your insights.